tagpr.go

```diff
@@ -619,15 +620,21 @@ func buildChunkSearchIssuesQuery(queryBase string, shasStr string) (chunkQueries
 		}
 		// Longer than 256 characters are not supported in the query.
 		// ref. https://docs.github.com/en/rest/reference/search#limitations-on-query-length
-		if len(query)+1+len(sha) >= 256 {
-			chunkQueries = append(chunkQueries, query)
-			query = queryBase
+		//
+		// However, although not explicitly stated in the documentation, the space separating
+		// keywords is counted as one or more characters, so it is possible to exceed 256
+		// characters if the text is filled to the very limit of 256 characters.
+		// For this reason, the maximum number of chars in the KEYWORD section is limited here to 200.
+		tempKeywords := append(keywords, sha)
+		if len(strings.Join(tempKeywords, " ")) >= 200 {
```

Comment:
Please add a constant and put this explanation aside.

It would help to understand the comment is about the value and not the algorithm

Reply:
I fixed it, how about this?